compiler/suggest.nim

```diff
@@ -265,8 +265,8 @@ proc fieldVisible*(c: PContext, f: PSym): bool {.inline.} =
       if fmoduleId == module.id: return true
     if f.kind == skField:
       var symObj = f.owner
-      if symObj.typ.kind in {tyRef, tyPtr}:
-        symObj = symObj.typ[0].sym
+      if symObj.typ.skipTypes({tyGenericBody}).kind in {tyRef, tyPtr}:
```

Comment:
What about `tyGenericInst`? Why not use `skipTypes(abstractInst)` here?

Reply:
I changed to `{tyGenericBody, tyGenericInst, tyGenericInvocation, tyAlias}` now to be consistent with `toObjectFromRefPtrGeneric`.